components/ProgressCard.tsx

```diff
@@ -1,93 +1,99 @@
-            fill="transparent"
-            stroke="#fff"
-            strokeDasharray={`${progressCount} ${100 - progressCount}`}
-            strokeDashoffset="20"
-            strokeWidth="4.5"
-          />
-          <g className="circle-label">
-            <text
-              color="#fff"
-              x={x}
-              y="23"
-              fill="#fff"
-              style={{ font: 'bold 10px sans-serif' }}
-            >
-              {progressCount}
-              <tspan fontSize="8px">%</tspan>
-            </text>
-          </g>
-        </svg>
-      )}
+        <g className="circle-label">
+          <text
+            color="#fff"
+            x={x}
+            y="23"
+            fill="#fff"
+            style={{ font: 'bold 10px sans-serif' }}
+          >
+            {progressCount}
+            <tspan fontSize="8px">%</tspan>
+          </text>
+        </g>
+      </svg>
     </div>
   )
 }
 
-const ProgressCard: React.FC<Props> = ({ progressCount }) => {
-  const setHeaderFromProgress = () => {
-    if (progressCount <= 13) {
-      // Has not finished 1st module yet
-      return `You're off to a great start!`
-    } else if (progressCount <= 50) {
-      return `Keep up the great work!`
-    } else if (progressCount < 100) {
-      return `Well done! You're over half way!`
-    } else {
-      // Assertion: progressCount = 100
-      return `Congratulations! You've finished the course!`
-    }
+const setHeaderFromProgress = (progressCount: number) => {
+  if (progressCount <= 13) {
+    // Has not finished 1st module yet
+    return `You're off to a great start!`
+  } else if (progressCount <= 50) {
+    return `Keep up the great work!`
+  } else if (progressCount < 100) {
+    return `Well done! You're over half way!`
+  } else {
+    // Assertion: progressCount = 100
+    return `Congratulations! You've finished the course!`
   }
+}
 
+const ProgressCard: React.FC<CardProps> = ({ progressCount, loggedIn }) => {
   return (
     <div
       className={`${styles['progress-card__container']} d-flex card shadow-sm mt-3 bg-primary text-white p-2 border-0`}
     >
       <div className="card-body">
-        <ProgressSVG progressCount={progressCount} />
-        <h4 className={`${styles['progress-card__title']} mt-3`}>
-          {setHeaderFromProgress()}
-        </h4>
+        {loggedIn ? (
+          <>
+            <ProgressSVG progressCount={progressCount} />
+            <h4 className={`${styles['progress-card__title']} mt-3`}>
+              {setHeaderFromProgress(progressCount)}
+            </h4>
+          </>
+        ) : (
+          <h4 className={`${styles['progress-card__title']} mt-3`}>
+            Join and start learning!
+          </h4>
+        )}
+
         <div className="mt-3">
-          Login to
           <NavLink
             className={`${styles['progress-newuser__chatlink']} text-white`}
-            path="https://chat.c0d3.com/login"
+            path="https://discord.gg/c0d3"
```

Comment:
Maybe you should update css for styling discord link. Previously link to mattermost was surrounded by text from each side so additional padding worked but right now it looks off. Or simply add link to `Discord` word instead of `Join us on Discord`. 

![margins](https://user-images.githubusercontent.com/3845012/116052305-287a6000-a682-11eb-95e9-04016bd39482.png)

Reply:
Thanks, I hadn't noticed it. I'll reduce the link to be just on `Discord` since there is a Discord card above the progress card anyway